queue/queue.go

```diff
@@ -0,0 +1,32 @@
+package party
+
+import (
+)
+
+type Queue struct {
+	 songList []string
+}
+
+func NewQueue() *Queue {
+	q := Queue{
+		songList:make([]string, 10),

```

Comment:
Yeah, I think you're right. We only ever need to have it sorted on the client, which should be pretty straight forwards. We can iterate over the whole thing to find maximums for "up next". 

Map operations are really straight forwards, maybe we should just use those for now